Ductus.FluentDocker.MsTest/Ductus.FluentDocker.MsTest.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks Condition="'$(OS)' != 'Unix'">netcoreapp1.1;netcoreapp2.0;net452</TargetFrameworks>
```

Comment:
Because you removed `netstandard1.6` from Ductus.FluentDocker you would need to remove `netcoreapp1.1`

Reply:
I have applied your proposed fix. Thanks a bunch! :)